### PR TITLE
Enforce no transactions before sending to notify queue

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: chroma_core/services/corosync/__init__.py [hunk 1]
```diff
@@ -156,7 +152,7 @@ def is_new(peer_node_identifier):
                     #  Raise an Alert - system suppresses duplicates
                     log.debug("Alert notify on %s: active=%s" % (host, not host_reported_online))
                     HostOfflineAlert.notify(host, not host_reported_online)
-                    if host_reported_online == False:
+                    if host_reported_online is False:
```
Comment (utopiabound) [inline chroma_core/services/corosync/__init__.py]: Isn't this change worse? (c.f. [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations))
```suggestion
                    if not host_reported_online:
```

Comment (jgrund) [reply chroma_core/services/corosync/__init__.py]: Flake8 seems to be indifferent:

screen shot 2019-01-18 at 9 39 12 am

I'm also indifferent, so I'll update.

Comment (jgrund) [reply chroma_core/services/corosync/__init__.py]: Though codefactor follows pep 8 and didn't like `is False`

**Comment by jgrund** [comment]
Not worth pursuing further.

**Pull request closed without merging**